### ISSUE TITLE
Failed to move Surveillance Station

### ISSUE THREAD

**Issue by blessendor** [007revad/Synology_app_mover#110]
Moving process was failed with error (didn't note it - something about application state), and Surveillance Station seems was moved to volume1 (as far I checked paths), but was displayed  as it is still installed on the volume2. Latter I found other apps like cloud sync was dissappeared from the app menu, and wrong old location. I tried reinstall (without uninstall) SS app from the spk, and this time moving was done. But I lost my camera setings and storage of its recordings was disassocitated from the camera. I hoped it will be reconnected after adding camera to SS with the same name, but it was not.

Restoring drives from the backup now (forgot to create a snapshot before moving).

p.s. during the first moving SS there was error like ' already exist folder/path /volume1/../lib/ffmpeg' on the destination volume (possible some old installation garbage). In the new SS this path is a symlink to /var/...

**Comment by 007revad** [comment]
Pity you didn't save a screenshot of the script's output.

**Comment by blessendor** [comment]
Yep, and restoring from my backup didn't help me - my SS settings lost. (i didn't restored from backup a volume where is surveillance share mounted - bcs it not included to backup). So after first login DSM asked me to Repair all my apps.
P.S. my dsm is virtual machine

**Comment by blessendor** [comment]
I still have some data on the volume2 (original instalation volume), and will try to copy/backup/swap from /volume2/@appstore/SurveillanceStation to /volume1/@appstore/SurveillanceStation

**Comment by slimline33** [comment]
I think he got this error

`You selected Surveillance Station in /volume1

Destination volume is /volume2

Checking size of Surveillance Station
Size of selected app(s) is 461 MB

Ready to Move Surveillance Station to /volume2? [y/n]
y

Copying /volume1/@appstore/SurveillanceStation to /volume2
Removing /volume1/@appstore/SurveillanceStation
Moving /volume1/@apptemp/SurveillanceStation to /volume2
cp: '/volume2/@appconf/SurveillanceStation' and '/volume2/@appconf/SurveillanceStation' are the same file
Line 583: ERROR Copying /volume2/@appconf/SurveillanceStation to /volume2 failed!
progstatus called from move_pkg_do line 847
`

**Comment by 007revad** [comment]
@slimline33 Which version of the script are you using?

**Comment by slimline33** [comment]
@007revad latest. 3.1.6.0

**Comment by slimline33** [comment]
@007revad but I see you changed something 4 days ago. Sorry, I can not try the newest version. Because I reinstalled it manually.

**Comment by 007revad** [comment]
> Line 583: ERROR Copying /volume2/@appconf/SurveillanceStation to /volume2 failed!
> progstatus called from move_pkg_do line 847

Those line numbers don't match v3.1.60

If you still have the syno_app_mover.sh file that you used can you run the script with the -v option which will just show the version.

**Comment by 007revad** [comment]
> Line 583: ERROR Copying /volume2/@appconf/SurveillanceStation to /volume2 failed!
> progstatus called from move_pkg_do line 847

@slimline33  Those line numbers don't match v3.1.60

If you still have the syno_app_mover.sh file that you used can you run the script with the -v option which will just show the version.
